x-pack/test/detection_engine_api_integration/security_and_spaces/tests/read_rules.ts

```diff
@@ -25,16 +25,250 @@ import {
 // eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext) => {
   const supertest = getService('supertest');
+  const spaces = getService('spaces');
+  const es = getService('es');
 
   describe('read_rules', () => {
     describe('reading rules', () => {
       beforeEach(async () => {
         await createSignalsIndex(supertest);
+        await spaces.create({ id: '714-space', name: '714-space' });
+
+        // migrated rule SO
+        await es.index({
+          id: 'alert:90e3ca0e-71f7-513a-b60a-ac678efd8887',
+          index: '.kibana',
+          refresh: true,
+          body: {
```

Comment:
I would move this into a separate test file and wire that up against the `index.ts` of tests. Name it something like "read_resolved_rules.ts" or "read_migrated_resolved_rules" or just something to remember for later to keep the resolved tests separate.